toctoc/slickPostgreSql/src/main/scala/io.buildo.toctoc/slick/authentication/PostgreSqlSlickTokenBasedRecoveryFlow.scala

```diff
@@ -0,0 +1,26 @@
+package io.buildo.toctoc
+package slick
+package authentication
+
+import core.authentication.TokenBasedRecovery.TokenBasedRecoveryFlow
+import token.PostgreSqlSlickAccessTokenAuthenticationDomain
+import login.PostgreSqlSlickLoginAuthenticationDomain
+
+import monix.catnap.FutureLift
+import cats.effect.Sync
+import _root_.slick.jdbc.JdbcBackend.Database
+
+import scala.concurrent.Future
+import java.time.Duration
+
+class PostgreSqlSlickTokenBasedRecoveryFlow[F[_]: Sync: FutureLift[?[_], Future]](
```

Comment:
we've decided to remove the specific flows from toctoc, so I wouldn't add this one.